ci/env/install-bazel.sh

```diff
@@ -98,24 +98,6 @@ bazel --version
 # clear bazelrc
 echo > ~/.bazelrc
 
-if [[ "${TRAVIS-}" == true ]]; then
-  echo "build --config=ci-travis" >> ~/.bazelrc
-
-  # If we are in Travis, most of the compilation result will be cached.
-  # This means we are I/O bounded. By default, Bazel set the number of concurrent
-  # jobs to the the number cores on the machine, which are not efficient for
-  # network bounded cache downloading workload. Therefore we increase the number
-  # of jobs to 50
-  # NOTE: Normally --jobs should be under 'build:ci-travis' in .bazelrc, but we put
-  # it under 'build' here avoid conflicts with other --config options.
-  echo "build --jobs=50" >> ~/.bazelrc
-fi
-
-if [[ "${GITHUB_ACTIONS-}" == "true" ]]; then
```

Comment:
is this because we are not using GHA anymore?

Reply:
nope, not anymore.